webview-ui/connectionsetup.js

```diff
@@ -241,6 +244,13 @@ function connectionCheckFailure(reason) {
   byId('connectionStatus').innerText = `Failed: ${reason}`;
 }
 
+function populateTokenField(token) {
+  byId('token').value = token;
+  byId('tokenStatus').innerText = 'Token Received!';
+  byId('tokenStatus').classList.remove('hidden');
+  byId('saveConnection').disabled = false;
```

Comment:
IMHO, this (c|sh)ould be a call to `toggleSaveConnectionButton`.

Reply:
Good point! done